common/src/main/java/org/opensearch/ml/common/input/MLInput.java

```diff
@@ -40,6 +42,12 @@ public class MLInput implements Input {
     public static final String INPUT_INDEX_FIELD = "input_index";
     public static final String INPUT_QUERY_FIELD = "input_query";
     public static final String INPUT_DATA_FIELD = "input_data";
+    // For trained model
+    public static final String RETURN_BYTES_FIELD = "return_bytes";
+    public static final String RETURN_NUMBER_FIELD = "return_number";
+    public static final String TARGET_RESPONSE_FIELD = "target_response";
+    public static final String TARGET_RESPONSE_POSITIONS_FIELD = "target_response_positions";
```

Comment:
These three fields requires description imo.

Reply:
Sure, will add some description in next PR.